File: src/main/java/com/giant/module/auth/vo/SignInVo.java
package com.giant.module.auth.vo;

import com.giant.module.auth.entity.type.AuthRole;
import com.giant.module.employee.entity.type.EmployeeRole;

public record SignInVo(
        Long accountId,
        String passwordHash,
        AuthRole authRole,
        String employeeCode,
        String employeeName,
        EmployeeRole employeeRole,
        String departmentName,
        String teamName,
        String positionName
) {
}

File: src/main/kotlin/com/giant/module/auth/dto/vo/SignInVo.kt
package com.giant.module.auth.dto.vo

import com.giant.module.auth.entity.type.AuthRole
import com.giant.module.employee.entity.type.EmployeeRole

data class SignInVo(
    val accountId: Long,
    val passwordHash: String,
    val authRole: AuthRole,
    val employeeCode: String,
    val employeeName: String,
    val employeeRole: EmployeeRole,
    val departmentName: String,
    val teamName: String,
    val positionName: String
)
